index.html

```diff
@@ -6,13 +6,26 @@
 	-->
 	<head>
 
-		<title><!-- your title goes here --></title>
+		<title>
+			<center>My Two Texts</center>
```

Comment:
The `<center>` HTML tag only works within the HTML `<body>`, not in the `<head>`